### PR TITLE
1510 Add find-a-partner page

### PR DESCRIPTION
**Pull request by solazio**
## Done
- Add `/partners/find-a-partner` page

## QA
- Go to [/partners/find-a-partner](http://0.0.0.0:8002/partners/find-a-partner) 
- Compare to [design](https://github.com/canonical-web-and-design/web-squad/issues/1510)

## Issue

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1510

### CODE REVIEW

File: static/sass/_pattern_buttons.scss [hunk 1]
```diff
@@ -1,4 +1,33 @@
 @mixin canonical-p-buttons {
+  .p-button--accordion {
+    background-color: inherit;
+    border: none;
+    background: {
+      position: top 50% left $sph-inner;
+      repeat: no-repeat;
+    }
+    background-color: inherit;
+    border: 0;
```
Comment (sowasred2012) [inline static/sass/_pattern_buttons.scss]: the `border` and `background-color` rules are repeated here, and I think the `background-color` rule could be merged into the `background` object on line 5.

File: templates/partners/find-a-partner.html [hunk 2]
```diff
@@ -0,0 +1,496 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <section class="p-strip--image is-dark is-deep u-no-padding--top u-no-padding--bottom">
```
Comment (sowasred2012) [inline templates/partners/find-a-partner.html]: I don't think you need the `is-deep` class, you're removing the padding it adds with the utility classes.

**Comment by anasereijo** [comment]
- [ ] search filters - when it's opened seems out of place. please align it to the left edge of the grid and make sure the width is the same as the page grid. (same issue on mobile)

Canonical _ The company behind Ubuntu

- [ ] search results: I searched for something and didn't get anything back. Isn't this working?

**Comment by solazio** [comment]
@anasereijo I aligned the filters. The functionality for search or filters has not been implemented yet. It will be a separate PR.

**Comment by anasereijo** [comment]
@solazio the filters seem to be open as default so I can't close it anymore.

not sure if possible but on mobile, can you please align the "+ filters" ?

Canonical _ The company behind Ubuntu (3)

**Comment by anasereijo** [comment]
perfect @solazio

**Pull request merged**